lib/schemaUtils.js

```diff
@@ -2891,6 +2912,20 @@ module.exports = {
         (pmSuffix[pMax - i].startsWith(':')) || // postman segment is a pathVar
         (pmSuffix[pMax - i].startsWith('{{') && pmSuffix[pMax - i].endsWith('}}')) // postman segment is an env var
       ) {
+        // for exact match increase fix matched segments count (used for determining order for multiple matches)
+        if (schemaPath[sMax - i] === pmSuffix[pMax - i]) {
+          fixMatchedSegments++;
+        }
+
+        // for variable match increase variable matched segments count (used for determining order for multiple matches)
```

Comment:
for `/a/b/{{c}}` and `/a/b/{{c}}`, is the fixMatchedSegments value supposed to be 2 or 3?
also, rename to 'fixedMatchedSegments` everywhere

Reply:
Yeah it would be 3 here, Moving `fixMatchedSegments` check after and in `else if` statements to `variableMatchedSegments` segment. so only one of them is valid per segment.